website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -18,6 +18,7 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## June 2025
 
+- **New**: The <Constant name="semantic_layer" /> now supports Trino as a data platform. For more details on how to set up the <Constant name="semantic_layer" /> for Trino, see [Set up the <Constant name="semantic_layer" />](/docs/use-dbt-semantic-layer/setup-sl).
```

Comment:
Small nit, but this reads like we're going to take them to a section that has information specific to Trino setup, when we're really only adding it to an existing list. Suggest a slightly broader stroke:
```suggestion
- **New**: The <Constant name="semantic_layer" /> now supports Trino as a data platform. For more details, see [Set up the <Constant name="semantic_layer" />](/docs/use-dbt-semantic-layer/setup-sl).
```